src/js/components/FileInput/FileInput.js

```diff
@@ -80,6 +81,7 @@ const FileInput = forwardRef(
     const [hover, setHover] = React.useState();
     const [dragOver, setDragOver] = React.useState();
     const aggregateThreshold = (multiple && multiple.aggregateThreshold) || 10;
+    const max = multiple && multiple.max;
```

Comment:
this is a bit awkward since you use it in numeric expressions and it could be boolean and such this way. How about something like `const max = multiple?.max || Number.POSITIVE_INFINITY`. You might be able to still default to false but that makes for some confusing logic where it's used.

Reply:
Adding `Number.POSITIVE_INFINITY` returns true everytime for conditons. This is for `Simple` story.
![grommet](https://user-images.githubusercontent.com/61521805/128430768-256cc5b5-5a23-4bf6-b9ff-30685f03ffa0.PNG)